File: loop.py
print(f'--------------------------');

top = 20000
expected = top * top;
print(f'Python: running up to {top} ({expected}) iterations');

total = 0;
for i in range(top):
    for j in range(top):
        total += 1
print(f'{"OK" if total == expected else "BAD"}')

File: loop.php
<?php

printf("--------------------------\n");

$top = 20000;
$expected = $top * $top;
printf("PHP: running up to %d (%d) iterations\n", $top, $expected);

$total = 0;
for ($i = 0; $i < $top; $i++) {
  for ($j = 0; $j < $top; $j++) {
    $total += 1;
  }
}
printf("%s\n", $total == $expected ? "OK" : "BAD");

?>
